llm_on_ray/inference/predictors/hpu_predictor.py

```diff
@@ -284,10 +292,6 @@ def load_model_and_tokenizer(self):
         self.world_size = int(os.environ["WORLD_SIZE"])
         self.local_rank = int(os.environ["LOCAL_RANK"])
         self.device = torch.device("hpu")
-        # optimize transformers for gaudi
-        from optimum.habana.transformers.modeling_utils import adapt_transformers_to_gaudi
-
-        adapt_transformers_to_gaudi()
```

Comment:
If this function is not executed in every worker, will it work as expected?

Reply:
Same as above, this function will be executed earlier.